metadata.rb

```diff
@@ -26,7 +26,7 @@
 Creates a swap file of the recommended size considering the system memory.
 EOS
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.3.0' # WiP
+version '0.3.1' # WiP
```

Comment:
Please don't change bump the version. The version 0.3.0 is still unreleased.

Reply:
Sorry